libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2521,6 +2523,15 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 	annotation.referencedDeclaration = possibleMembers.front().declaration;
 	annotation.type = possibleMembers.front().type;
 
+	if (
+		auto const& funType = dynamic_cast<FunctionType const*>(exprType) &&
```

Comment:
```suggestion
		dynamic_cast<FunctionType const*>(exprType) &&
```
Actually you don't need the variable

Reply:
I do not need it at all, I figured that out when I posted PR, and fixed already